udata/models/__init__.py

```diff
@@ -103,6 +103,7 @@ def resolve_model(self, model):
 def init_app(app):
     if app.config['TESTING']:
         app.config['MONGODB_DB'] = '{MONGODB_DB}-test'.format(**app.config)
+        app.config['MONGODB_PORT'] = 27018
```

Comment:
I think this will only cover the case for those using the embedded docker file (and for circleci tests). Some are using a native standalone mongo installation.
Maybe we can define some `MONGODB_TEST_*` settings allowing for those who needs it to run on the same DB (or maybe the opposite, allowing in our case to define the alternate db only we used)